src/test/resources/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder6.java

```diff
@@ -1,3 +1,14 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.SuppressWarningsHolder
+aliasList = (default)
+
+
+com.puppycrawl.tools.checkstyle.checks.whitespace.TypecastParenPad
+option = space
```

Comment:
`com.puppycrawl.tools.checkstyle.checks.whitespace.TypecastParenPad` -> `com.puppycrawl.tools.checkstyle.checks.whitespace.TypecastParenPadCheck`

Reply:
@jxr98 Please don't resolve review items on your own. Reply 'Done', the reviewer will check it and resolve the review item.

Reply:
Okay.